learns-app-content/week-11/objectives.md

```diff
@@ -1,23 +1,25 @@
 <!-- h1, h2 already used by CTD Learns -->
 ### Topics Covered
 
-- topic 1
-- topic 2
-- topic n
+- Polishing a front app for your portfolio
```

Comment:
@rebekahcallkacz - I have conflicting wording in my notes - is this wording what you meant or should the focus on overall the portfolio?